packages/core/src/shareback/metadata/constants.ts

```diff
@@ -0,0 +1,29 @@
+/*
+CONFIDENTIALITY_CODE is always N, or normal, indicating its normal PHI
+FORMAT_CODE is always urn:ihe:iti:xds:2017:mimeTypeSufficient, which is sufficient for all document types.
```

Comment:
Can we associate these comments to the respective field?

Reply:
will do